### DISCUSSION TITLE
How To create table through flutter code?

### DISCUSSION THREAD

**Discussion by SamarpanCoder2002** [supabase/supabase#2981] [Questions] [answered]
I want to create a table using flutter code. But not find anythings to proceed with. Please help me

**Comment by silentworks** [comment, marked as answer]
Tables are created either using the Supabase UI or using an external Postgres client application on your computer. Unlike NoSQL, in relational databases you create your tables and schemas in the database before you start working with the tables.

**Reply by SamarpanCoder2002** [reply]
I want to make a chat application, where every user contains a table.. for that when new users register in the app... I have to create the table in supabase for himself/herself manually from flutter code only... not from supabase UI or external Postgres client... Please help me.

**Reply by silentworks** [reply]
Yet again this is not how relational databases work. You need to shift your thinking from document based databases to relational databases. Inside the Supabase dashboard there is an example slack app which you can try out, you can follow that to learn how to create a chat application.

**Reply by SamarpanCoder2002** [reply]
> Yet again this is not how relational databases work. You need to shift your thinking from document based databases to relational databases. Inside the Supabase dashboard there is an example slack app which you can try out, you can follow that to learn how to create a chat application.

ok trying
